generals/game.py

```diff
@@ -56,27 +56,24 @@ def __init__(self, map: np.ndarray, agents: List[str]):
         ##########
         # Spaces #
         ##########
-        box = gym.spaces.Box(low=0, high=1, shape=spatial_dim, dtype=np.float32)
+        grid_multi_binary = gym.spaces.MultiBinary(spatial_dim)
         self.observation_space = gym.spaces.Dict(
             {
                 "army": gym.spaces.Box(
                     low=0, high=np.inf, shape=spatial_dim, dtype=np.float32
                 ),
```

Comment:
Shouldn't `dtype` be `int` or something like that? Similarly in numerous other places like `test_game.py`, `rendering.py`, and in this file?

Reply:
I'll think about it. The thing is that this value is then used for ML frameworks who convert it back to `float32` anyway. Probably it is a good convention to make it int.